ansible/provision-basic-cluster.yml

```diff
@@ -15,3 +15,15 @@
       ansible.builtin.include_role:
         name: redpanda.cluster.redpanda_broker
       when: not skip_node | default(false) | bool
+
+- name: Provision redpanda console
+  hosts: client
+  vars:
+    advertise_public_ips: true
+    redpanda_version: latest
+    install_console: true
```

Comment:
Is there a PR to add the usage of this flag to our documentation repo?

Reply:
Good call

Reply:
For other reviewers, here is the PR: https://github.com/redpanda-data/documentation-private/pull/1942

Thanks Gene :+1: